bats/tests/k8s/helm-install-rancher.bats

```diff
@@ -7,6 +7,7 @@ setup() {
     # If sudo permissions are not present, these tests should be skipped.
     if is_linux; then
         run sudo sysctl -w net.ipv4.ip_unprivileged_port_start=443
+        assert_nothing
```

Comment:
Given the comment above, wouldn't we can something like

```
if [ $status -ne 0 ]; then
  … set something to skip stuff
```

Reply:
Yes, but I don't want to make other changes in this PR, except "fixing" the new linting warnings